torchvision/transforms/functional_pil.py

```diff
@@ -121,7 +126,13 @@ def adjust_gamma(img, gamma, gain=1):
 
 
 @torch.jit.unused
-def pad(img, padding, fill=0, padding_mode="constant"):
+def pad(
+    img: Image.Image,
+    padding: Union[int, List[int], Tuple[int, ...]],
+    fill: Optional[Union[float, List[float], Tuple[float, ...]]] = 0,
```

Comment:
This is defined as int on the tensor equivalent method, though it supports both. Do we want them aligned?

Reply:
I think we want float in tensor equivalent method to align this.

Reply:
no strong opinion here. I leave it up to you.